src/test/java/teammates/test/cases/datatransfer/AdminEmailAttributesTest.java

```diff
@@ -259,26 +259,24 @@ public void testSanitizeForSaving() {
 
     @Test
     public void testSendDateForDisplay() {
-        Calendar calendar = formatDateForAdminEmailAttributesTest(validAdminEmailAttributesObject.sendDate);
-        String expectedDate = TimeHelper.formatTime12H(calendar.getTime());
+        validAdminEmailAttributesObject.sendDate = Instant.now();
```

Comment:
Is this change in the logic of the test intentional? The original test does not set this field. Ok, I see that this is for consistency with the test below.

Reply:
The original test coupled it together with `formatDateForAdminEmailAttributesTest`, which I believe was a mistake.